osu.Game.Rulesets.Catch/Mods/CatchModFloatingFruits.cs

```diff
@@ -0,0 +1,23 @@
+﻿using osu.Framework.Graphics;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Screens.Play;
+
+namespace osu.Game.Rulesets.Catch.Mods
+{
+    public class CatchModFloatingFruits : Mod, IApplicableToPlayer
+    {
+        public override string Name => "Floating Fruits";
+        public override string Acronym => "FF";
+        public override string Description => "The fruits are... floating?";
+        public override double ScoreMultiplier => 1;
+        public override IconUsage? Icon => FontAwesome.Brands.Fly;
+
+        public void ApplyToPlayer(Player player)
+        {
+            player.DrawableRuleset.Anchor = Anchor.Centre;
+            player.DrawableRuleset.Origin = Anchor.Centre;
+            player.DrawableRuleset.Scale = new osuTK.Vector2(1, -1);
```

Comment:
This implementation, however simple, causes the catcher-following combo counter to appear upside down on legacy skins.

Reply:
Not sure how much of an issue this one is to be honest. The "Barrel Roll" mod does the same for SPM counter and judgements, for instance. Maybe adds to the style?

Reply:
It bothers me less in barrel roll, possibly because in that case motion is constant and continuous. It looks weirder here where everything is statically flipped.

I guess if you think it's fine I'm not going to insist on having this fixed, just figured it'd be something people will bring up. Could also be left as a follow up.

Reply:
That said I do think fixing this shouldn't be too hard if going the minimal effort way (get combo counter via `ChildrenOfType()`, check if it's legacy, if it is, flip Y axis - the playfield flip will cancel out the combo counter flip and it'll be upright again).

Reply:
Generally trying to keep use of `ChidrenOfType` to tests only (i think we have so far?) so if we do decide to account for flipping it'd need to be exposed in a better way.